modules/mobile/app/helpers/mobile/appointments_helper.rb

```diff
@@ -17,7 +17,7 @@ def create_new_appointment(params)
 
     def backfill_location(appointment)
       unless appointment[:clinic].nil?
-        clinic = vaos_mobile_facility_service.get_clinic(station_id: [:location_id], clinic_id: appointment[:clinic])
+        clinic = vaos_mobile_facility_service.get_clinic(appointment[:location_id], appointment[:clinic])
```

Comment:
Is this change related to our form?

Reply:
No. The branch is updated with main. It is now removed.